packages/expo-contacts/android/src/main/java/expo/modules/contacts/ContactsModule.kt

```diff
@@ -126,7 +126,7 @@ class ContactsModule : Module() {
     get() = appContext.permissions ?: throw Exceptions.PermissionsModuleNotFound()
 
   private val activity: Activity
-    get() = appContext.activityProvider?.currentActivity ?: throw Exceptions.MissingActivity()
+    get() = appContext.currentActivity ?: throw Exceptions.MissingActivity()
```

Comment:
could we maybe move that exception to the `appContext.currentActivity` getter?

Reply:
Something like this?
```kotlin
override val currentActivity: Activity
  get() {
    val activity = activityProvider?.currentActivity
      ?: (reactContext as? ReactApplicationContext)?.currentActivity
    
    return activity ?: throw Exceptions.MissingActivity()
  }
```

Reply:
I like it, now we don't need that boilerplate :)

Reply:
I like it, but perhaps we need to introduce a different getter for that. We can't change the current one. In some places, we actually check if the activity is null or not.

Reply:
Any suggestions for what we would should name a new property if we don't change this one? 
```kotlin
val activity: Activity
  get() {
    val current = activityProvider?.currentActivity
      ?: (reactContext as? ReactApplicationContext)?.currentActivity
    
    return current ?: throw Exceptions.MissingActivity()
  }
```